src/modules/json.c

```diff
@@ -0,0 +1,142 @@
+// JSON Parser Events
+const char json_typename[][64] = {
+    [JSON_ERROR]      = "com.domeengine.json:JSON_ERROR",
```

Comment:
I like this style of specifying errors. Wren and DOME's error handling isn't particularly powerful, but this sort of provides a structure.